src/cli.js

```diff
@@ -65,10 +73,31 @@ async function loadPlugin(name) {
   return plugins;
 }
 
+function findPkgJsonConfig() {
+  // fallback to original config name 'wsk' in case anything goes wrong
+  // for safe backwards compatibility
+  let cfg = 'wsk';
+
+  try {
+    // find package.json in parent directories (like yargs does)
+    const pkg = escalade(process.cwd(), (_, names) => (names.includes('package.json') ? 'package.json' : undefined));
+
+    if (pkg) {
+      const pkgJson = JSON.parse(readFileSync(pkg, 'utf8'));
+      // search for config key in package.json by priority in PKG_CONF array
+      cfg = PKG_CONF.find((key) => key in pkgJson) || cfg;
+    }
+  } catch (e) {
+    // ignore
+  }
+
+  return cfg;
+}
+
 export default class CLI {
   buildArgs(plugins) {
     this._yargs = yargs()
-      .pkgConf('wsk')
+      .pkgConf(findPkgJsonConfig())
```

Comment:
Unfortunately yargs does not support multiple config names here. Hence I have to find the package.json first (in the same manner as yargs, walking up the tree from current dir) and see what of our candidate names is present in the file. And then tell yargs to look for that one. If none is present or things fail, we just default to `wsk` so it all behaves as before.